spec/models/inventory_spec.rb

```diff
@@ -52,11 +52,13 @@
   end
 
   describe '#save' do
-    subject {
-      create(:inventory)
-    }
+    let(:owner) { FactoryGirl.create(:user) }
+    subject { FactoryGirl.create(:inventory, owner: owner) }
```

Comment:
This seems to be testing the behavior of the factory more than the model.  It will likely shake out, but if the factory does fail, we'll get the failure not just here, but everywhere that uses it.